homework.md

```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript specification.A programming language used on the internet.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1.Helpes us to share information.
+2.Enables people to make thier own changes to that information,and share it back.
+3.It allows  us to track how inforamtion has changed between each checkpoint and to go back to the older versions of the work if we to see what we tried before or to undo changes.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+The "clone" command downloads an existing Git repository to your local computer.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+git log lists the commits made in that repository in reverse chronological order.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+When you made a change on GitHub, it not only saved the change and committed the file at the same time.
 
```

Comment:
Hi Lawrence, I just want to confirm, do you mean that when you push  - it saves changes and commits? maybe we should just check the steps that need to be taken before you "push"

Reply:
Noted. I've checked the steps